test/functional/inabox/test-inabox-messaging-host.js

```diff
@@ -403,4 +404,51 @@ describes.realWin('inabox-host:messaging', {}, env => {
       expect(host.getFrameElement_(sourceMock, sentinel)).to.be.null;
     });
   });
+
+  describe('unregisterIframe', () => {
+    it('unregisters frames', () => {
```

Comment:
nit: this test is kinda hard to look at. maybe add some line breaks or comments to separate setup / test / expectation verification?

Reply:
Done